src/app/InteractionModelEngine.cpp

```diff
@@ -413,6 +413,61 @@ CHIP_ERROR InteractionModelEngine::ParseAttributePaths(const Access::SubjectDesc
     return err;
 }
 
+CHIP_ERROR InteractionModelEngine::ParseEventPaths(const Access::SubjectDescriptor & aSubjectDescriptor,
```

Comment:
Put another way: this behavior of doing ACL checks is specified for DoS protection, but if it can be worked around by just using a wildcard path instead of a concrete path, it's not providing any such protection.